src/shared/components/checkbox/Checkbox.scss

```diff
@@ -5,31 +5,27 @@
   grid-template-columns: 27px 1fr;
 }
 
-.defaultCheckbox {
+.checkboxDefault {
   cursor: pointer;
   justify-self: center;
-  height: 14px;
-  width: 14px;
-  border: 1px solid $grey;
-  background-color: $white;
   transition: background-color 0.1s ease-in-out;
   margin-top: 7px;
+  background-color: var(--checkbox-background-color, #{$white});
+  border: 1px solid var(--checkbox-border-color, #{$grey});
+  height: var(--checkbox-dimension-size, 13px);
+  width: var(--checkbox-dimension-size, 13px);
```

Comment:
Isn't "dimension-size" excessive? `--checkbox-side`? `--checkbox-side-length`?

Reply:
If we use `--checkbox-side-length` then how about the other dimension (width or height)? Also isn't `--checkbox-side` ambiguous?

Reply:
> how about the other dimension (width or height)

Well, our hope with checkboxes is that they are squares. If they are squares (you can confirm with Andrea whether they are), then their height should equal their width, and we should only expose a single variable that controls this size.

`--checkbox-side-length` or `--checkbox-side-size` look ok.

Reply:
That makes sense. I'll check with Andrea if we might have any outliers for checkboxes.

Reply:
I've changed this to `--checkbox-side-size` ✅